generate_certs.sh

```diff
@@ -1,15 +1,15 @@
-#!/usr/bin/env bash
```

Comment:
this is the way we should probably do it, as bash might not exist in `/usr/bin/bash` but asking `/usr/bin/env` for the `bash` path works on spicy configs.